### ISSUE TITLE
Bug template

### ISSUE THREAD

**Issue by eddiejaoude** [TransformCore/documentation-templates#22]
Have a template for when someone is logging a bug

**Comment by rorywoodward** [comment]
Are we assuming Jira?

**Comment by eddiejaoude** [comment]
Can we keep it generic?

**Comment by rorywoodward** [comment]
Yes - that would be nice, but different bug tracking solutions will likely have different mandatory fields. Perhaps we should have a template with basic must-have info, then a series of nice-to-haves (preferably with benefits of including, and effects of excluding). We can then say that when a new bug-tracking solution is used for the first time, the relevant project is responsible for providing a mapping in this repo, and justifying exclusions?

**Comment by eddiejaoude** [comment]
> different bug tracking solutions will likely have different mandatory fields

I was thinking more for the **Description** field, which then can put in to various fields as required?

> template with basic must-have info, then a series of nice-to-haves

Sounds good :+1:

**Comment by eddiejaoude** [comment]
My initial thoughts:

Required:
* Screenshot
* Steps to reproduce
* Browser
* Environment *(dev, test, demo, prod etc)*
* Application version

Nice to have:
* BDD test
